src/software/thunderscope/robot_diagnostics/chicker.py

```diff
@@ -90,32 +171,14 @@ def change_button_state(self, button, enable):
 
     def refresh(self):
 
-        # slider values
-        geneva_value = self.geneva_slider.value()
-        self.geneva_label.setText(Slot.Name(geneva_value))
-
-        power_value = self.power_slider.value()
-        self.power_label.setText(str(power_value))
-
-        # if autokick is enabled, we don't want to allow kick/chip
-        auto_kick_enabled = (
-            self.auto_kick_button.isChecked() or self.auto_chip_button.isChecked()
-        )
-        self.change_button_state(self.kick_button, not auto_kick_enabled)
-        self.change_button_state(self.chip_button, not auto_kick_enabled)
-
-        power_control = PowerControl()
-        power_control.geneva_slot = geneva_value
+        # refreshes button state based on enable boolean
+        self.change_button_state(self.kick_button, self.kick_button_enable)
+        self.change_button_state(self.chip_button, self.chip_button_enable)
 
         # If auto is enabled, we want to populate the autochip or kick message
         if self.auto_kick_button.isChecked():
-            power_control.chicker.auto_chip_or_kick.autokick_speed_m_per_s = power_value
+            self.send_kick_or_chip(True)
```

Comment:
When reading this function its not clear whether `True` refers to kick or chip. following up with the last comment can we instead use an Enum something like `ChickerCommandMode` with options for KICK, CHIP, AUTOKICK and AUTOCHIP?